Readme.md

```diff
@@ -0,0 +1,5 @@
+To build locally
+
+jekyll serve --watch
+
+(you need to install jekyll first)
```

Comment:
It removed the offending line in the `main.scss` which was importing the `minima` theme (which is the default theme anyway) and it seems to work as before, hopefully it will work for you as well.